### DISCUSSION TITLE
UTM Guest OS Issue

### DISCUSSION THREAD

**Discussion by sameoj** [utmapp/UTM#5127] [UTM Help (macOS)]
Hi,

I`m trying to login to my Window Server installed on UTM.

Each time I press these keys: Command+Option+Esc; it displays "Force Quit Application." Please see attached screenshot.

Can anyone, please, tell me what I`m doing wrong?

Thank you in advance
![Screenshot 2023-03-13 at 4 03 19 PM (3)](https://user-images.githubusercontent.com/127625165/225781978-7dbb55c7-154e-4044-b883-c2aaec0748f6.png)

**Comment by harlleysathler** [comment]
Cmd + Opt + ESC is the global macOS shortcut to kill programs. I think you would try Ctrl+Opt+Fn+Delete to get the Windows login.

**Reply by sameoj** [reply]
Thanks, it worked.
